elements/lisk-chain/src/block_header.ts

```diff
@@ -179,6 +179,95 @@ export class BlockHeader {
 		}
 	}
 
+	public validateGenesis(): void {
+		const header = this._getBlockHeaderProps();
+		const errors = validator.validate(blockHeaderSchema, header);
+
```

Comment:
I think check for `The value b.header.previousBlockID can be any 32-byte value.` is missing